apps/src/lib/util/StatsigReporter.js

```diff
@@ -125,6 +129,19 @@ class StatsigReporter {
     }
   }
 
+  findOrCreateStableId() {
+    const stableIdKey = 'statsig_stable_id';
```

Comment:
Nit: If this is a string constant, can it be defined up by the other constants iin lines 15-16 please

Reply:
Yes def!